modules/monitor.php

```diff
@@ -31,7 +31,16 @@ public function jetpack_modules_loaded() {
 		Jetpack::enable_module_configurable( $this->module );
 	}
 
+	/**
+	 * Send an API request to check if the monitor is active.
+	 *
+	 * @return mixed
+	 *
+	 * @deprecated 4.2.0 The method is not being used, to be removed.
```

Comment:
even though they have not been used since 4.2, I'm not sure we should retroactively deprecate them... There's a (small) chance someone out there in the wild is using them... I think we can add a note that they've been obsolete for a while, but the version where they got deprecate should reflect the "truth" -> 8.9. What do you think?

Reply:
I do agree that we shouldn't retroactively deprecate them. It can be confusing for us in the future when we try to gauge how much time has passed since they've been deprecated.

I would suggest adding a line in the inline doc, to Leo's point, that it has been unused since Jetpack 4.2 and officially deprecated in Jetpack 8.9.

Reply:
Fixed, thanks for your input team.